include/a3/ui_f/hpp/defineResinclDesign.inc

```diff
@@ -328,19 +328,19 @@
 #define IDC_RSCDISPLAYRESPAWN_MAP		1050
 
 //--- RscDisplaySelectIsland
-// #define IDC_RSCDISPLAYSELECTISLAND_TITLE		1000
-// #define IDC_RSCDISPLAYSELECTISLAND_NAME		1002
-// #define IDC_RSCDISPLAYSELECTISLAND_AUTHOR		1003
-// #define IDC_RSCDISPLAYSELECTISLAND_NAMEBACKGROUND	1004
-// #define IDC_RSCDISPLAYSELECTISLAND_ISLANDPANORAMA	1200
-// #define IDC_RSCDISPLAYSELECTISLAND_INTERLACING	1201
-// #define IDC_RSCDISPLAYSELECTISLAND_CLOUD1		1202
-// #define IDC_RSCDISPLAYSELECTISLAND_CLOUD2		1203
-// #define IDC_RSCDISPLAYSELECTISLAND_CLOUD3		1204
-// #define IDC_RSCDISPLAYSELECTISLAND_CLOUD4		1205
-// #define IDC_RSCDISPLAYSELECTISLAND_ISLANDGROUP	2300
-// #define IDC_RSCDISPLAYSELECTISLAND_BUTTONCONTINUE	2600
-// #define IDC_RSCDISPLAYSELECTISLAND_BUTTONCANCEL	2700
```

Comment:
This was commented out due to duplicates in BI defines which HEMTT does warn about.